pkg/controller/replica.go

```diff
@@ -91,6 +91,11 @@ func (r *ReconcileReplica) Reconcile(ctx context.Context, request reconcile.Requ
 		if volumeDetachRequested(azVolumeAttachment) {
 			switch azVolumeAttachment.Status.State {
 			case azdiskv1beta2.DetachmentFailed:
+				// if detachment failed and the driver is uninstalling, delete azvolumeattachment CRI to let uninstallation proceed
+				if r.isDriverUninstall() {
+					_ = r.cachedClient.Delete(ctx, azVolumeAttachment)
```

Comment:
It would probably be good to emit a log to indicate that the replica attachment CRI is being deleted without detachment.

Reply:
Use `workflow.GetObjectDetails` to include `AzVolumeAttachment` details, e.g.

```go
r.logger.Info("Deleting AzVolumeAttachment without detaching disk", workflow.GetObjectDetails(azVolumeAttachment)...)
```